### PR TITLE
Add Dutch UI translation

### PR DESCRIPTION
**Pull request by Willem3141**
## Summary of changes

I noticed that there are GUI translations in Plover, so I went ahead and made a Dutch translation:

image

### Pull Request Checklist
- [ ] Changes have tests *(not applicable I think?)*
- [x] News fragment added in news.d. See [documentation](https://github.com/openstenoproject/plover/blob/master/doc/developer_guide.md#making-a-pull-request) for details

### CODE REVIEW

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 1]
```diff
@@ -0,0 +1,1021 @@
...
+"Door te arpeggiëren is het mogelijk om een\n"
+"niet-NKRO-toetsenbord te gebruiken.\n"
+"\n"
+"Iedere toets kan één voor één ingedrukt worden,\n"
+"waarna de spatiebalk de aanslag verzendt."
+
+#. Widget: “MainWindow”, tooltip.
+#: plover/gui_qt/main_window_ui.py:172 plover/gui_qt/main_window_ui.py:212
+msgid "Disconnect and reconnect the machine."
+msgstr "Verbreek en herstel vervolgens de verbinding met de machine."
```
Comment (evils) [inline plover/messages/nl/LC_MESSAGES/plover.po:748]: "vervolgens" lijkt een beetje overbodig

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 2]
```diff
@@ -0,0 +1,1021 @@
...
+
+#. Widget: “MainWindow”, shortcut.
+#: plover/gui_qt/main_window_ui.py:194
+msgid "Ctrl+Q"
+msgstr "Ctrl+Q"
+
+#. Widget: “MainWindow”, text.
+#: plover/gui_qt/main_window_ui.py:196
+msgid "&Configure"
+msgstr "&Instellingen"
```
Comment (evils) [inline plover/messages/nl/LC_MESSAGES/plover.po:798]: "&Instellen"?
het is niet "&Configurations"
(ik ken de context niet, "&settings" kan bedoelt zijn)

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 3]
```diff
@@ -0,0 +1,1021 @@
...
+msgstr ""
+"Plover is een vrij open-source programma met het doel om realtime\n"
+"stenografische technologie beschikbaar te maken niet alleen voor\n"
+"stenografen, maar ook voor hackers, hobbyisten, fans van toegankelijkheid\n"
+"en snelheidsduivels."
+
+#. Widget: “AboutDialog”, windowtitle.
+#: plover/gui_qt/about_dialog_ui.py:38
+msgid "Plover: About"
+msgstr "Plover: Over"
```
Comment (evils) [inline plover/messages/nl/LC_MESSAGES/plover.po:68]: :smile:

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 4]
```diff
@@ -0,0 +1,1021 @@
...
+msgid "Add translation dialog opacity:"
+msgstr "Doorzichtigheid van vertaling-toevoegen-venster:"
+
+#: plover/gui_qt/config_window.py:303
+msgid ""
+"Set the translation dialog opacity:\n"
+"- 0 makes the dialog invisible\n"
+"- 100 is fully opaque"
+msgstr ""
+"Dit stelt in hoe doorzichtig het venster om vertalingen toe te voegen is:\n"
```
Comment (evils) [inline plover/messages/nl/LC_MESSAGES/plover.po:271]: "vertalingsvenster"?

**Review by evils — COMMENTED** [review]
negeer dit gerust, ik ben vlaams en gebruik vooral engels

(feel free to ignore this, my dutch ain't great)

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 5]
```diff
@@ -0,0 +1,1021 @@
...
+"Ted Morin\n"
+"Benoit Pierre\n"
+"\n"
+"en nog vele anderen op GitHub:\n"
+"<https://github.com/openstenoproject/plover>"
+
+#. Short description for Plover, currently not used in the interface.
+#: plover/__init__.py:33
+msgid "Open Source Stenography Software"
+msgstr "Open Source-software voor stenografie"
```
[the remark this answers is not in the document]
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:52]: Hmm, this is always difficult with English loans... https://nl.wikipedia.org/wiki/Opensourcesoftware writes it as a single word, so I'll follow that.

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 6]
```diff
@@ -0,0 +1,1021 @@
...
+
+#. Widget: “MainWindow”, shortcut.
+#: plover/gui_qt/main_window_ui.py:200
+msgid "Ctrl+,"
+msgstr "Ctrl+,"
+
+#. Widget: “MainWindow”, text.
+#: plover/gui_qt/main_window_ui.py:202
+msgid "Open config &folder"
+msgstr "Open instellingenmap"
```
[the remark this answers is not in the document]
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:813]: Yes, nice catch.

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 7]
```diff
@@ -0,0 +1,1021 @@
...
+#: plover/gui_qt/dictionary_editor_ui.py:116
+msgid "Del"
+msgstr "Del"
+
+#. Widget: “DictionariesWidget”, text.
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionaries_widget_ui.py:122
+#: plover/gui_qt/dictionary_editor_ui.py:118
+msgid "&Undo"
+msgstr "Ongedaan maken"
```
[the remark this answers is not in the document]
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:590]: Also nice catch :slightly_smiling_face:

File: plover/messages/nl/LC_MESSAGES/plover.po (hunk 1 above)
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:748]: Yeah, this one is slightly awkward. Removed "vervolgens".

File: plover/messages/nl/LC_MESSAGES/plover.po (hunk 2 above)
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:798]: This one makes more sense in context:
image

File: plover/messages/nl/LC_MESSAGES/plover.po (hunk 4 above)
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:271]: Nice!

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 8]
```diff
@@ -0,0 +1,1021 @@
...
+
+#. Widget: “DictionaryEditor”, text.
+#: plover/gui_qt/dictionary_editor_ui.py:112
+msgid "&Delete"
+msgstr "&Verwijderen"
+
+#. Widget: “DictionaryEditor”, tooltip.
+#: plover/gui_qt/dictionary_editor_ui.py:114
+msgid "Delete selected entries."
+msgstr "Verwijder de geselecteerde regels."
```
[the remark this answers is not in the document]
Comment (Willem3141) [reply plover/messages/nl/LC_MESSAGES/plover.po:706]: I could make it "regel(s)" or something, but I think that is overkill?

[the remark this answers is not in the document]
Comment (evils) [reply plover/messages/nl/LC_MESSAGES/plover.po:706]: i think "Verwijder geselecteerde regels." works

File: plover/messages/nl/LC_MESSAGES/plover.po [hunk 9]
```diff
@@ -0,0 +1,1021 @@
...
+msgid "Show suggestions:"
+msgstr "Suggesties weergeven:"
+
+#: plover/gui_qt/config_window.py:300
+msgid "Open the suggestions dialog on startup."
+msgstr "Open het venster met suggesties bij het opstarten."
+
+#: plover/gui_qt/config_window.py:301
+msgid "Add translation dialog opacity:"
+msgstr "Doorzichtigheid van vertaling-toevoegen-venster:"
```
Comment (evils) [inline plover/messages/nl/LC_MESSAGES/plover.po:263]:
```suggestion
msgstr "Doorzichtigheid van het vertalingsvenster:"
```

**Comment by benoit-pierre** [comment]
Thanks @Willem3141 & @evils, I'll go ahead and merge.

**Pull request merged**